x-hooks/use-network.tsx

```diff
@@ -79,7 +78,7 @@ export function useNetwork() {
         }
 
         if (queryChainName) {
-          const data = rows[0];
+          const data = rows.find((network) => network?.chain?.chainShortName === queryChainName)
```

Comment:
I think you can use `toLowerCase` for both variables to normalize the values, just to ensure that it will work even if someone type the chain name with upper case characters